jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -163,7 +163,7 @@ private Decoder createDecoder(ObjectMapper objectMapper) {
                 .randomGenerator(new Random())
                 .state(new ThreadLocalServerClientAndLocalSpanState(
                         getIpAddress(), 0 /** Client TCP port. */, userAgent))
-                .spanCollector(new SlfLoggingSpanCollector("ClientTracer(" + userAgent + ")"))
+                .spanCollector(new SlfLoggingSpanCollector("tracing.client." + userAgent))
```

Comment:
the user agent string can be pretty wild, cf. https://github.com/palantir/http-remoting/blob/develop/jaxrs-clients/src/test/java/com/palantir/remoting1/jaxrs/feignimpl/UserAgentInterceptorTest.java#L58